### PR TITLE
Fixed unable to download files while comparing products #3501

### PR DESCRIPTION
**Pull request by devansh-webkul**
**BUGS:**
Issue: https://github.com/bagisto/bagisto/issues/3501

### CODE REVIEW

File: packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php [hunk 1]
```diff
@@ -111,6 +111,10 @@ class="material-icons cross fs16"
                                                             : '{{ __('velocity::app.shop.general.no') }}'"
                                                 ></span>
                                                 @break;
+                                            @case('file')
+                                                <a v-if="product.product['{{ $attribute['code'] }}']" :href="`${$root.baseUrl}/storage/${product.product['{{ $attribute['code'] }}']}`" class="fs16">Download</a>
```
Comment (rahulcs0082) [inline packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php:115]: Kindly write a translations.

File: packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php [hunk 2]
```diff
@@ -111,6 +111,10 @@ class="material-icons cross fs16"
...
+                                                <a v-else class="fs16">No File Available!</span>
```
Comment (rahulcs0082) [inline packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php:116]: same here.

File: packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php (hunk 1 above)
Comment (devansh-webkul) [reply packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php]: added

File: packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php (hunk 2 above)
Comment (devansh-webkul) [reply packages/Webkul/Velocity/src/Resources/views/shop/guest/compare/compare-products.blade.php]: added

**Comment by bhanu-webkul** [comment]
uploaded file should come in place of download - https://prnt.sc/tr8l73

**Comment by vaishaliwebkul** [comment]
@devansh-webkul Kindly add the downloaded icon along with file name https://prnt.sc/ttum32

**Comment by vaishaliwebkul** [comment]
working fine https://prnt.sc/tuaia2

**Pull request merged**